pkg/controller/manager/manager_controller.go

```diff
@@ -264,183 +289,208 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
+	// Package up the request parameters needed to reconcile
+	req := Request{
+		instance:       instance,
+		NamespacedName: request.NamespacedName,
+		variant:        variant,
+		installation:   installation,
+		license:        license,
+	}
+
+	return r.reconcileInstance(ctx, logc, req)
+}
+
+func (r *ReconcileManager) reconcileInstance(ctx context.Context, logc logr.Logger, request Request) (reconcile.Result, error) {
+	certificateManager, err := certificatemanager.Create(r.client, request.installation, r.clusterDomain, request.TruthNamespace())
 	if err != nil {
-		r.status.SetDegraded(operatorv1.ResourceCreateError, "Unable to create the Tigera CA", err, reqLogger)
+		r.status.SetDegraded(operatorv1.ResourceCreateError, "Unable to create the Tigera CA", err, logc)
 		return reconcile.Result{}, err
 	}
 
 	// Get or create a certificate for clients of the manager pod es-proxy container.
-	svcDNSNames := append(dns.GetServiceDNSNames(render.ManagerServiceName, render.ManagerNamespace, r.clusterDomain), "localhost")
+	svcDNSNames := append(dns.GetServiceDNSNames(render.ManagerServiceName, request.InstallNamespace(), r.clusterDomain), "localhost")
 	tlsSecret, err := certificateManager.GetOrCreateKeyPair(
 		r.client,
 		render.ManagerTLSSecretName,
-		common.OperatorNamespace(),
+		request.TruthNamespace(),
 		svcDNSNames)
 	if err != nil {
-		r.status.SetDegraded(operatorv1.ResourceReadError, "Error getting or creating manager TLS certificate", err, reqLogger)
+		r.status.SetDegraded(operatorv1.ResourceReadError, "Error getting or creating manager TLS certificate", err, logc)
 		return reconcile.Result{}, err
 	}
 
 	// Determine if compliance is enabled.
-	complianceLicenseFeatureActive := utils.IsFeatureActive(license, common.ComplianceFeature)
+	complianceLicenseFeatureActive := utils.IsFeatureActive(request.license, common.ComplianceFeature)
 	complianceCR, err := compliance.GetCompliance(ctx, r.client)
 	if err != nil && !errors.IsNotFound(err) {
-		r.status.SetDegraded(operatorv1.ResourceReadError, "Error querying compliance: ", err, reqLogger)
+		r.status.SetDegraded(operatorv1.ResourceReadError, "Error querying compliance: ", err, logc)
 		return reconcile.Result{}, err
 	}
 
 	// Build a trusted bundle containing all of the certificates of components that communicate with the manager pod.
 	// This bundle contains the root CA used to sign all operator-generated certificates, as well as the explicitly named
 	// certificates, in case the user has provided their own cert in lieu of the default certificate.
-	trustedSecretNames := []string{
-		render.PacketCaptureCertSecret,
-		monitor.PrometheusTLSSecretName,
-		relasticsearch.PublicCertSecret,
-		render.ProjectCalicoAPIServerTLSSecretName(installation.Variant),
-		render.TigeraLinseedSecret,
+	trustedSecretNames := []render.Secret{
+		render.PacketCaptureServerCert(r.multiTenant, request.InstallNamespace()),
```

Comment:
Trying out an approach here. The secrets that we fetch may be in one of two namespaces:

- 1: single-tenant, will be in `tigera-operator`
- 2: multi-tenant, will be in the tenant's namespace (which varies for each call to Reconcile())

So, this approach turns the constant into a function, that takes the install namespace and returns a struct instead of a string.... (We can actually remove the first argument from this function which makes it simpler).